genomedata/__init__.py

```diff
@@ -494,6 +453,131 @@ def vars(self):
             square(self.means)
 
 
+class _ChromosomeList(object):
+    def __init__(self):
+        self.open_chromosomes = {}  # NB: maintained by subclass
+
+    def __getitem__(self, name):
+        """Returns an existing opened chromosome if it exists"""
+        res = None
+        # If we have opened the chromosome already
+        if (name in self.open_chromosomes and
+           self.open_chromosomes[name].isopen):  # NB: manual closure check
```

Comment:
I think there's an indent issue here

Reply:
`visually indented line with same indent as next logical line - flake8(E129)` is often what happens in this case with multiple clauses enclosed by paranthesis. The `if (` is a perfect 4 space line-up for the following line which apparently `flake8` does not like